src/XamlStyler/Options/StylerOptions.cs

```diff
@@ -356,6 +356,29 @@ public string ConfigPath
         [Browsable(false)]
         public bool SuppressProcessing { get; set; }
 
+        private string endOfLine;
+
+        [Category("Misc")]
+        [RefreshProperties(RefreshProperties.All)]
+        [Description("Defines end of line character. Specify 'lf' or 'crlf'; otherwise, default character of the host will be used.")]
+        [JsonProperty("EndOfLine", DefaultValueHandling = DefaultValueHandling.IgnoreAndPopulate)]
```

Comment:
For [consistency with Visual Studio](https://learn.microsoft.com/en-us/visualstudio/ide/encodings-and-line-breaks?view=vs-2022), can we call this "LineEndings"?